### ISSUE TITLE
downloadnetfilerawdata fails

### ISSUE THREAD

**Issue by mikeubell** [caciviclab/disclosure-backend#70]
(env)MacBook-Pro:~/disclosure-backend master
$ python disclosure-backend/manage.py downloadnetfilerawdata
/Users/ubell/disclosure-backend/env/src/django-calaccess-raw-data/calaccess_raw/**init**.py:3: RemovedInDjango19Warning: The utilities in django.db.models.loading are deprecated in favor of the new application loading system.
  from django.db.models.loading import get_models, get_app

Traceback (most recent call last):
  File "disclosure-backend/manage.py", line 9, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/ubell/disclosure-backend/env/lib/python2.7/site-packages/django/core/management/**init**.py", line 338, in execute_from_command_line
    utility.execute()
  File "/Users/ubell/disclosure-backend/env/lib/python2.7/site-packages/django/core/management/**init**.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/ubell/disclosure-backend/env/lib/python2.7/site-packages/django/core/management/base.py", line 390, in run_from_argv
    self.execute(_args, *_cmd_options)
  File "/Users/ubell/disclosure-backend/env/lib/python2.7/site-packages/django/core/management/base.py", line 441, in execute
    output = self.handle(_args, *_options)
  File "netfile/management/commands/downloadnetfilerawdata.py", line 94, in handle
    self.connect2 = Connect2API()
  File "netfile/connect2_api.py", line 56, in **init**
    self.api = SwaggerClient(CONNECT2_SPEC)
  File "/Users/ubell/disclosure-backend/env/src/swaggerpy-master/swaggerpy/client.py", line 199, in **init**
    self.api_docs = loader.load_resource_listing(url_or_resource)
  File "/Users/ubell/disclosure-backend/env/src/swaggerpy-master/swaggerpy/swagger_model.py", line 172, in load_resource_listing
    self.process_resource_listing(resource_listing)
  File "/Users/ubell/disclosure-backend/env/src/swaggerpy-master/swaggerpy/swagger_model.py", line 196, in process_resource_listing
    processor.apply(resources)
  File "/Users/ubell/disclosure-backend/env/src/swaggerpy-master/swaggerpy/processors.py", line 118, in apply
    self.process_operation(**context.args)
  File "/Users/ubell/disclosure-backend/env/src/swaggerpy-master/swaggerpy/swagger_model.py", line 68, in process_operation
    validate_required_fields(operation, required_fields, context)
  File "/Users/ubell/disclosure-backend/env/src/swaggerpy-master/swaggerpy/swagger_model.py", line 212, in validate_required_fields
    "Missing fields: %s" % ', '.join(missing_fields), context)
swaggerpy.processors.SwaggerError: ('Missing fields: httpMethod', ParsingContext(stack=['resources=https://netfile.com/Connect2/api/resources', u'resource=https://netfile.com/Connect2/api/resource/auth', 'api=/auth', 'operation=Authenticate']), None)

**Comment by ebrensi** [comment]
yes I ran into this too.  I thought it was just me.

**Comment by rkiddy** [comment]
Is this from a fresh clone of the repo? This kind of error has happened to me a few times when I did a pull, got updated content in an existing clone, and then ran the migrate command with existing data.

If I get a fresh clone and run through the procedure, then there has been no problem. And I copy in the TSV and other data tables from a backup and this has not caused a problem and I have not needed to download all the data.

Having the migrate work from an alpha state is necessary and that has worked. It has not been as reliable to migrate from an intermediate update, but this might not be as critical to fix.
